test_scripts/API/CancelInteraction/003_SendCancelInteraction_ScrollableMessage_HMIReceivesRequest.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) app1 is registered on SDL
+--
+--  Steps:
+--  1) app1 sends a sends an ScrollableMessage RPC
+--  2) app1 sends a CancelInteraction Request with the functionID of ScrollableMessage
+--  3) the HMI receives the CancelInteraction Request and replies
+--
+--  Expected:
+--  1) app1 receives SUCCESS from the CancelInteraction
+--  2) app1 receives ABORTED from the ScrollableMessage
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variables ]]
+local rpcInteraction = {
+  name = "ScrollableMessage",
+  hmi_name = "UI.ScrollableMessage",
+  params = {
+    scrollableMessageBody = "big scrollable\n\t\tmessage"
```

Comment:
Add `cancelID`

Reply:
added in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2206/commits/471e8b5b7364595f4a5d0d0cb371f70f2d31a730